crates/cxx-qt-gen/test_outputs/inheritance.cpp

```diff
@@ -20,3 +20,5 @@ MyObject::MyObject(QObject* parent)
   , ::rust::cxxqtlib1::CxxQtLocking()
 {
 }
+
+#include "moc_inheritance.cpp"
```

Comment:
Note these files are just test outputs for the test inputs. Eg the expected C++ output for the given C++ input.

Should these have explicit moc includes ? As we would then need to add this to the generation phase which then needs to know the resultant file name.

Or should these be added to the `cxx-qt-lib` sources instead ?

Reply:
ok I didn't know that it was not compiled.